Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Views
+    private var dayLabel: UILabel!
+    private var backCircle: UIView!
+
+    // MARK: - Info
+    /// Weekday this cell represents
+    var day: WeekDay?
+    /// Is this day today
+    private var today: Bool = false
```

Comment:
I think the comment is a bit redundant. Let's remove the comment and rename the variable to `isToday` so that the naming is clear enough